content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+is found in `.git/refs/exps`, and earlier ones are in its [reflog].
+
+[reflog]:
+  https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefreflogareflog
+
+</details>
+
+```dvc
+$ dvc exp run --queue -S units=10
+Queued experiment '1cac8ca' for future execution.
+$ dvc exp run --queue -S units=64
+Queued experiment '23660bb' for future execution.
+$ dvc exp run --queue -S units=128
+Queued experiment '3591a5c' for future execution.
+$ dvc exp run --queue -S units=256
+Queued experiment '4109ead' for future execution.
+```
+
+> `dvc exp show` will mark queued experiments with an asterisk `*`.
+
+Each experiment is derived from the workspace at the time it's queued. If you
+make changes in the workspace afterwards, they won't be reflected in queued
+experiments (once run).
+
+<details>
+
+### How are queued experiments isolated? (Temporary directories)
+
+To guarantee that queued experiments derive from their original workspace, DVC
+creates a copy of it in `.dvc/tmp/exps/`, where the experiment will run. All
+these workspaces share the main project <abbr>cache</abbr>.
+
+If you want to isolate an experiments this way without queuing it, you can use
+the `--temp` option. This allows you to continue working while a long experiment
+runs.
+
+```dvc
+$ nohup dvc exp run --temp &
+[1] 30473
+nohup: ignoring input and appending output to 'nohup.out'
+```
+
+> The above example creates a `nohup.log` file in the original workspace with
+> the output of the DVC process.
+
+Note that Git-ignored files/dirs are explicitly excluded from queued/temp runs
+to avoid committing unwanted files into final ([persisted]) experiments.
+
+[persisted]: /doc/user-guide/experiment-management#persistent-experiments
+
+</details>
+
+Run them all one-by-one with the `--run-all` flag. The order of execution is
+independent of their creation order.
+
+```dvc
+$ dvc exp run --run-all
+```
+
+> ⚠️ Note that only tracked files and directories will be included in
+> `--queue/temp` experiments. To include untracked files, stage them with
+> `git add` first (before `dvc exp run`) and `git reset` them afterwards.
+
+To remove all experiments from the queue and start over, you can use
+`dvc exp remove --queue`. 📖 See [Cleaning-Up Experiments][clean-up] for
+details.
+
+[clean-up]: /doc/user-guide/experiment-management/cleaning-up-experiments
+
+### Running experiments in parallel
+
+DVC allows to run queued experiments in parallel by specifying a number of
+execution processes (`--jobs`):
+
+```dvc
+$ dvc exp run --run-all --jobs 4
+```
+
+> Note that since each experiment runs in an independent temporary directory,
+> common <abbr>stages</abbr> across experiment pipelines are always run
+> separately even if they share param values or other dependencies.
+
+⚠️ Parallel runs are experimental and may be unstable at this time. ⚠️ Make sure
+you're using a number of jobs that your environment can handle (no more than the
+CPU cores).
+
+## Checkpoint experiments
```

Comment:
What's the plan for this section? It's a nice summary, but I see it as a comment in the checkpoints guide and as a section of this guide.

Reply:
This is basically pulled from https://dvc.org/doc/command-reference/exp/run#checkpoints. We need to determine what to leave on the cmd ref (after this PR). There's also a task about reviewing the whole checkpoints guide in https://github.com/iterative/dvc.org/issues/2548.

Reply:
p.s. please leave this comment unresolved.